docs/website/content/v0.6/en/guides/local/registry-cache.md

```diff
@@ -0,0 +1,88 @@
+---
+title: Registry Cache
+---
+
+In this guide we will create a set of local caching Docker registry proxies to minimize local cluster startup time.
+
+When running Talos locally, pulling images from Docker registries might take significant amount of time.
+We spin up local  caching pass-through registries to cache images and configure local Talos cluster to use those proxies.
+Similar approach might be used to run Talos in production in air-gapped environments.
+It can be also used to verify that all the images are available in local registries.
+
+## Requirements
+
+The follow are requirements for creating set of caching proxies:
+
+- Docker 18.03 or greater
+- Local cluster requirements for either [docker](docker) or [fireracker](firecracker).
+
+## Launch the Caching Docker Registry Proxies
+
+With default configuration, Talos pulls from `docker.io`, `k8s.gcr.io` and `quay.io`.
+If your configuration is different, you might need to modify the commands below:
+
+```bash
+docker run -d -p 5000:5000 \
+    -e REGISTRY_PROXY_REMOTEURL=https://registry-1.docker.io \
+    --restart always \
+    --name registry-docker.io registry:2
```

Comment:
```suggestion
    --name registry-docker.io registry:2.5
```

Reply:
`:2` should be fine here (we can use newer version, because upstream registry supports modern schema)

in fact, `:2` is higher version than `:2.5` (that is confusing, but that's how things are)